guides/micronaut-microservices-services-discover-eureka/bookinventory/groovy/src/test/groovy/example/micronaut/BooksControllerSpec.groovy

```diff
@@ -1,46 +1,40 @@
 package example.micronaut
 
 import io.micronaut.http.HttpRequest
-import io.micronaut.http.HttpStatus
+import io.micronaut.http.HttpResponse
 import io.micronaut.http.client.HttpClient
 import io.micronaut.http.client.annotation.Client
 import io.micronaut.http.client.exceptions.HttpClientResponseException
 import io.micronaut.test.extensions.spock.annotation.MicronautTest
+import jakarta.inject.Inject
 import spock.lang.Specification
 
-import jakarta.inject.Inject
+import static io.micronaut.http.HttpStatus.NOT_FOUND
+import static io.micronaut.http.HttpStatus.OK
 
 @MicronautTest
 class BooksControllerSpec extends Specification {
 
     @Inject
     @Client("/")
-    HttpClient client
+    HttpClient httpClient
 
     void "for a book with inventory true is returned"() {
         when:
-        Boolean hasStock = client.toBlocking().retrieve(HttpRequest.GET("/books/stock/1491950358"), Boolean)
-
-        then:
-        noExceptionThrown()
-        hasStock
-    }
-
-    void "for a book without inventory false is returned"() {
```

Comment:
why did we remove a test ?

Reply:
to be consistent with the Java and Kotlin code